manifest-reference.md

```diff
@@ -0,0 +1,357 @@
+
+*Example:*
+
+```toml
+maintainer = "jane.doe@example.com"
+```
+
+
+## Project author
+
+Information on the project author.
+
+*Example:*
+
+```toml
+author = "Jane Doe"
+```
+
+
+## Project copyright
+
+A statement clarifying the copyright status of the project.
+
+*Example:*
+
+```toml
+copyright = "Copyright 2020 Jane Doe"
+```
+
+
+## Project description
+
+The decription provides a short summary on the project.
+It should be plain text and not using any markup formatting.
+
+*Example:*
+
+```toml
+description = "A short summary on this project"
+```
+
+
+## Project categories
+
+The project can be associated with different categories.
+
+*Example:*
+
+```toml
+categories = ["io"]
+```
+
+
+## Project keywords
+
+The keywords field is an array of strings describing the project.
+
+*Example:*
+
+```toml
+keywords = ["hdf5", "mpi"]
+```
+
+
+## Project homepage
+
+URL to the webpage of the project.
+
+*Example:*
+
+```toml
+homepage = "https://stdlib.fortran-lang.org"
+```
+
+
+## Project targets
+
+Every fpm project can define library, executable and test targets.
+Library targets are exported and useable for other projects.
+
+
+### Library configuration
+
+Defines the exported library target of the project.
+A library is generated if the source directory found in a project.
+The default source directory is ``src`` but can be modifed in the *library* section using the *source-dir* entry.
+Paths for the source directory are given relative to the project root and use ``/`` as path separator on all platforms.
+
+*Example:*
+
+```toml
+[library]
+source-dir = "lib"
+```
+
+#### Custom build script
+
+> Supported in Bootstrap fpm only
+
+Projects with custom build scripts can specify those in the *build-script* entry.
+The custom build script will be executeted when the library build step is reached.
+
+*Example:*
+
+```toml
+[library]
+build-script = "build.sh"
+```
+
+Build scripts written in ``make`` are automatically detected and executed with ``make``
+
+```toml
+[library]
+build-script = "Makefile"
+```
+
+
+### Executable targets
+
+Executable targets are Fortran programs defined as *executable* sections.
+If no executable section is specified the ``app`` directory is searched for program definitions.
+For explicitly specified executables the *name* entry must always be specified.
+The source directory for each executable can be adjusted in the *source-dir* entry.
+Paths for the source directory are given relative to the project root and use ``/`` as path separator on all platforms.
+The source file containing the program body can be specified in the *main* entry.
+
+Executables can have their own dependencies.
+See [specifying dependencies](#specifying-dependencies) for more details.
+
+*Example:*
+
+```toml
+[[executable]]
+name = "app-name"
+source-dir = "prog"
+main = "program.f90"
+
+[[executable]]
+name = "app-tool"
+[executable.dependencies]
+helloff = { git = "https://gitlab.com/everythingfunctional/helloff.git" }
+```
+
```

Comment:
I've just realised that I didn't add support for `[executable.dependencies]` or `[test.dependencies]` in #193.
```suggestion
> The [executable.dependencies] syntax is supported in bootstrap fpm only
```